pandas/tseries/tests/test_resample.py

```diff
@@ -361,6 +361,23 @@ def test_fillna(self):
         with self.assertRaises(ValueError):
             r.fillna(0)
 
+    def test_fill_value(self):
+        # setup
+        rng = pd.date_range('1/1/2016', periods=10, freq='2S')
+        ts = pd.Series(np.arange(len(rng)), index=rng)
+        ts2 = pd.Series(np.random.rand(len(rng)), index=rng)
```

Comment:
can you add a test also to where DataFarme.asfreq is tested (prob not very much :<)

```
{lamb} grin ".asfreq" pandas\tests\
pandas\tests\frame\test_timeseries.py:
  299 :     def test_asfreq(self):
  300 :         offset_monthly = self.tsframe.asfreq(offsets.BMonthEnd())
  301 :         rule_monthly = self.tsframe.asfreq('BM')
  305 :         filled = rule_monthly.asfreq('B', method='pad')  # noqa
  309 :         filled_dep = rule_monthly.asfreq('B', method='pad')  # noqa
  313 :         result = zero_length.asfreq('BM')
  316 :     def test_asfreq_datetimeindex(self):
  320 :         df = df.asfreq('B')
  323 :         ts = df['A'].asfreq('B')
pandas\tests\indexes\test_datetimelike.py:
  830 :                 idx.get_loc(idx[1].asfreq('H', how='start'), method), 1)
  888 :         idx = pd.period_range('2000-01-01', periods=3).asfreq('H', how='start')
pandas\tests\plotting\test_datetimelike.py:
  319 :         bts = tm.makeTimeSeries().asfreq('BM')
  740 :         ts2 = ts.asfreq('T').interpolate()
pandas\tests\series\test_datetime_values.py:
   37 :         ok_for_period_methods = ['strftime', 'to_timestamp', 'asfreq']
pandas\tests\series\test_timeseries.py:
  407 :     def test_asfreq(self):
  411 :         daily_ts = ts.asfreq('B')
  412 :         monthly_ts = daily_ts.asfreq('BM')
  415 :         daily_ts = ts.asfreq('B', method='pad')
  416 :         monthly_ts = daily_ts.asfreq('BM')
  419 :         daily_ts = ts.asfreq(BDay())
  420 :         monthly_ts = daily_ts.asfreq(BMonthEnd())
  423 :         result = ts[:0].asfreq('M')
```

and pls add a tests for series as well